index.js

```diff
@@ -1,120 +1,100 @@
+const cache = require('@actions/tool-cache');
 const core = require('@actions/core');
 const { exec } = require('@actions/exec');
 const fs = require('fs');
 
-// Find all 'content-*.json' files in the directory. dirname should include the full path
```

Comment:
Why did this all get moved? Generally, it's best practice to have functions defined before they are used in JS.

Reply:
I just prefer the main function up top for readability with these procedural style scripts. I'm not married to it though, so I can change it back.